test/build/core-flags/core-flags.test.js

```diff
@@ -231,6 +239,22 @@ describe("core flags", () => {
             expect(stdout).toContain(`devtool: 'source-map'`);
         });
 
+        it("should allow string value devtool option using alias", async () => {
+            const { exitCode, stderr, stdout } = await run(__dirname, ["-d", "source-map"]);
+
+            expect(exitCode).toBe(0);
+            expect(stderr).toBeFalsy();
+            expect(stdout).toContain(`devtool: 'source-map'`);
+        });
+
+        it("should allow string value devtool option using alias #1", async () => {
+            const { exitCode, stderr, stdout } = await run(__dirname, ["-dsource-map"]);
```

Comment:
maybe you meant?
```suggestion
            const { exitCode, stderr, stdout } = await run(__dirname, ["-d=source-map"]);
```

Reply:
No, it is valid, commander supports this syntax

Reply:
ah nice lgtm then